ass1-sass/css/style.css

```diff
@@ -2274,6 +2274,71 @@
     .sidebar .sidebar-options .sidebar-logout i {
       margin: 0 10px; }
 
+.container {
+  background-color: #eee;
+  float: left;
+  height: 700px;
+  margin-left: 260px;
+  padding: 5px;
+  width: 82%; }
+  .container .container-info {
+    background-color: #fff;
+    line-height: 40px;
+    padding: 10px;
+    width: 100%; }
+    .container .container-info i {
+      font-size: 24px;
+      margin: 0 10px; }
+    .container .container-info h3 {
+      display: inline-block; }
+      .container .container-info h3 a {
+        color: #5C8BD3;
+        text-decoration: underline; }
+  .container .report-wrapper {
+    background-color: #fff;
+    margin: 5px 0;
+    padding: 10px;
+    width: 100%; }
+    .container .report-wrapper .report-left {
+      width: 50%; }
+      .container .report-wrapper .report-left .report-button {
+        background-color: #6B3A2B; }
+      .container .report-wrapper .report-left .report-subheading {
+        color: #5C8BD3;
+        font-size: 14px;
+        margin: 5px; }
+      .container .report-wrapper .report-left .report-img {
+        text-align: center;
+        width: 100%; }
+        .container .report-wrapper .report-left .report-img img {
+          width: 150px; }
+  .container .report-right {
+    width: 50%; }
+    .container .report-right .subscribe-button {
+      color: #FF9478; }
+    .container .report-right .form-wrapper {
+      margin: 10% 5% 5%;
+      /* .input-field input:focus + .email-label {
+        color: $light-blue;
+      }
+*/ }
+      .container .report-right .form-wrapper input[type=text] {
+        border-bottom: 1px solid #ccc;
+        display: block;
+        line-height: 20px;
+        margin: 20px 0;
+        padding: 20px 0;
+        width: 80%; }
+      .container .report-right .form-wrapper .email-label {
+        -webkit-transform: translateY(-100%);
+        transform: translateY(-100%); }
+      .container .report-right .form-wrapper .input-field input:focus {
+        border-bottom: 1px solid #5C8BD3; }
+      .container .report-right .form-wrapper .flat-subscribe-button {
+        color: #5C8BD3; }
+      .container .report-right .form-wrapper .add-button {
+        background-color: #4742B7; }
+
 body,
 html {
   font-family: 'Roboto', sans-serif;
```

Comment:
Then which font I should add... I didn't understand what you are trying to say here